src/foam/u2/view/ReferenceView.js

```diff
@@ -18,15 +18,22 @@
 foam.CLASS({
   package: 'foam.u2.view',
   name: 'ReferenceView',
-  extends: 'foam.u2.view.ChoiceView',
+  extends: 'foam.u2.view.RichChoiceView',
```

Comment:
Should these changes be part of this PR?

Reply:
:| yup right as usual - removed